package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "video-carousel-service",
+  "version": "1.0.0",
+  "description": "FEC Team Luke - videos carousel module",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --watch",
+    "start:dev": "nodemon server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hrr43fec-luke/video-carousel-service.git"
+  },
+  "author": "Priscila Gonzalez",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hrr43fec-luke/video-carousel-service/issues"
+  },
+  "homepage": "https://github.com/hrr43fec-luke/video-carousel-service#readme",
+  "dependencies": {
+    "@babel/preset-react": "^7.8.3",
```

Comment:
Minor: this should be a dev dependency I think

Reply:
Thanks!

Reply:
Thanks!